core/src/main/java/hudson/util/MaskingClassLoader.java

```diff
@@ -23,11 +23,9 @@
  */
 package hudson.util;
 
+import java.io.IOException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
```

Comment:
Generally prefer explicit to star imports, but I think we have no official style guide for the project…